frontend/.eslintrc.cjs

```diff
@@ -10,5 +10,6 @@ module.exports = {
   plugins: ['react-refresh'],
   rules: {
     'react-refresh/only-export-components': 'warn',
-  },
+    "@typescript-eslint/no-unused-vars": ["warn", { "ignoreRestSiblings": true }],
```

Comment:
This is so that we can remove the `node` parameter in EdtiableCode as part of the object spread. Otherwise when we do

In `{node, ...props}` the `props` is considered a "rest parameter", i.e. it contains all of the fields of the object except for `node`, which we don't want to be injecting onto the elements.

By default eslint will mark `node` as unused and throw a warning, this flags that node should not be marked unused b/c there is another rest parameter in the spread